### PR TITLE
Feature/price history

### PR DESCRIPTION
**Pull request by marcoalfonso**

### CODE REVIEW

File: apps/orams/components/ProviderList/ProviderList.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        </header>
+        <article role="main">
+          <div className="row">
+            {categories &&
+              categories.map((category, id = uniqueID()) =>
+                <div key={id} className="col-xs-12 col-sm-5">
+                  <div className={styles.categoryTitle}>
+                    {category.name}
+                  </div>
+                  {category.suppliers.map((supplier, subId = uniqueID()) =>
```
Comment (matt-sm) [inline apps/orams/components/ProviderList/ProviderList.js]: why not user `supplier.code` as the key?

File: apps/orams/components/PriceHistory/PriceHistory.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import { loadBuyerSuppliers, loadStepTwo, setStep, loadPriceData } from 'orams/actions/priceHistoryActions'
+
+class PriceHistory extends Component {
+  static propTypes = {}
+
+  componentDidMount() {
+    this.props.loadBuyerSuppliers()
+  }
+
+  loadStepTwo = supplierCode => {
```
Comment (matt-sm) [inline apps/orams/components/PriceHistory/PriceHistory.js]: This is confusing - `loadStepTwo` is also imported?

File: apps/orams/components/ProviderHistoryForm/ProviderHistoryForm.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+import SubmitForm from 'shared/form/SubmitForm'
+import StatefulError from 'shared/form/StatefulError'
+import ErrorBox from 'shared/form/ErrorBox'
+
+import styles from './ProviderHistoryForm.scss'
+
+class ProviderHistoryForm extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
```
Comment (matt-sm) [inline apps/orams/components/ProviderHistoryForm/ProviderHistoryForm.js]: any reason for initialising `this.state`?

File: apps/orams/components/ProviderList/ProviderList.js (hunk 1 above)
Comment (marcoalfonso) [reply apps/orams/components/ProviderList/ProviderList.js]: This is just a pattern we have been using all over.

Comment (matt-sm) [reply apps/orams/components/ProviderList/ProviderList.js]: But using `math.random` means id will different on each mount?  We do have stable keys like `supplier.code` which I thought was what FB recommend

Comment (marcoalfonso) [reply apps/orams/components/ProviderList/ProviderList.js]: Can I merge this?

**Pull request merged**